docs/obs/find_observations.rst

```diff
@@ -0,0 +1,66 @@
+.. _obs_find_observations:
+
+Find observations
+=================
+
+TODO: reference observation list/table format
+
+The ``gammapy-find-obs`` command line tool can be used to select a
+subset of observations from a given observation list.
+
+It works with `fits` files as input/output.
+
+Examples:
```

Comment:
Make this a sub-section, so that it gets a URL one can link to from emails or other parts of the Sphinx docs.
```
Examples
-------------
```